pennylane/interfaces/execution.py

```diff
@@ -618,74 +618,76 @@ def cost_fn(params, x):
             )(tapes)
         )
 
+    # the default execution function is batch_execute

```

Comment:
We should remove the changes from the execute pipeline, we will come back for another PR later.

Reply:
Fair point! Updated.